### ISSUE TITLE
[Feature]: Static DB Schema

### ISSUE THREAD

**Issue by jcbannon** [sqlitebrowser/sqlitebrowser#3449]
### Describe the new feature

Keep the DB Schema as set when executing SQL. In the schema, I display the fields, adjust the Name column, but upon each execution, this is collapsed.

### Do you have a screenshot? Please add screenshots to help explain your idea.

Before:
![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/11354896/c1ab3eb3-c8d2-49b6-b342-ab6e9d875833)

After:
![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/11354896/be1aad91-8978-4e39-bfc4-70278e41c54a)

**Comment by scottfurry** [comment]
If I'm understanding this correctly, we shouldn't collapse the trees on refresh. This may be a function of the Qt control we are using. The fix for us may involve _remember_ tree state before refreshing the control. We'll have to _play_ with this one to figure out.

**Comment by mgrojo** [comment]
Indeed, it wouldn't be easy to implement. @jcbannon, as a workaround, you could press * on Tables to expand all tables.
